tvm_binding/grouped_gemm_fp8.cu

```diff
@@ -0,0 +1,161 @@
+#include <dlpack/dlpack.h>
+
+#include <flashinfer/cutlass_utils.cuh>
+#include <flashinfer/gemm/group_gemm_fp8_groupwise_sm100.cuh>
+
+#include "tvm_binding_utils.h"
+
+__global__ void simple_print_kernel(void* data, int dtype_code) {
+  if (threadIdx.x == 0 && blockIdx.x == 0) {
+    if (dtype_code == kDLBfloat) {
+      // bfloat16
+      uint16_t* bf16_data = static_cast<uint16_t*>(data);
+      uint32_t full = ((uint32_t)bf16_data[0]) << 16;
+      float val = *reinterpret_cast<float*>(&full);
+      printf("GPU: D[0] = %.6f\n", val);
+    } else {
+      // float32
+      float* f32_data = static_cast<float*>(data);
+      printf("GPU: D[0] = %.6f\n", f32_data[0]);
+    }
+  }
+}
```

Comment:
Need to remove this given it was for debugging purpose.

Reply:
Let's clean up all of them after #1641  :)